doc/source/tune/user-guide.rst

```diff
@@ -194,6 +192,21 @@ You can restore a single trial checkpoint by using ``tune.run(restore=<checkpoin
         config={"env": "CartPole-v0"},
     )
 
+Distributed Checkpointing
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On a multinode cluster, Tune automatically creates a copy of all trial checkpoints on the head node. This requires the Ray cluster to be started with the :ref:`cluster launcher <ref-automatic-cluster>` and also requires rsync to be installed.
+
+Note that you must use the ``tune.checkpoint_dir`` API to trigger syncing.
+
+If you do not use the cluster launcher, you should set up a NFS or global file system and
+disable cross-node syncing:
+
+.. code-block:: python
+
+    tune.run(func, sync_to_driver=False)
+
+
```

Comment:
Maybe mention that Kubernetes users have to specify the KubernetesSyncer explicitly?

Reply:
done!